edkrepo/common/ui_functions.py

```diff
@@ -57,9 +57,9 @@ def print_info_msg(info_msg, header=True, force_color_output=None):
     """
     init_color_console(force_color_output)
     if header:
-        info_msg_formatted = "{}{}Info: {}{}{}".format(Style.BRIGHT, Fore.CYAN, Style.RESET_ALL, Fore.CYAN, info_msg)
+        info_msg_formatted = "{}{}Info: {}{}{}".format(Style.BRIGHT, Fore.WHITE, Style.RESET_ALL, Fore.WHITE, info_msg)

```

Comment:
Lets remove the color part of the formatting from this entirely and let it default to the users console settings.